app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -57,29 +58,33 @@ public function postRegister(ProfileRequest $request, User $user)
             $state_code = $location['iso_code'].'-'.$location['state'];
             $state = getStateByCode($state_code);
             $user->state = $state['id'];
-            $password = \Hash::make($request->input('password'));
-            $user->country = $request->input('country');
-            $user->password = $password;
+            $password = Str::random(20);
+            $user->password = \Hash::make($password);
             $user->town = $location['city'];
             $user->profile_pic = '';
             $user->active = 0;
             $user->mobile_verified = 0;
+            $user->country = $request->input('country');
             $user->mobile = ltrim($request->input('mobile'), '0');
             $user->mobile_code = $request->input('mobile_code');
             $user->role = 'user';
-            $user->first_name = strip_tags($request->input('first_name'));
-            $user->last_name = strip_tags($request->input('last_name'));
             $user->company = strip_tags($request->input('company'));
             $user->address = strip_tags($request->input('address'));
             $user->email = strip_tags($request->input('email'));
+            $user->first_name = strip_tags($request->input('first_name'));
+            $user->last_name = strip_tags($request->input('last_name'));
             $user->user_name = strip_tags($request->input('email'));
-
             $user->manager = $user->assignSalesManager();
             $user->ip = $location['ip'];
             $user->timezone_id = getTimezoneByName($location['timezone']);
             $user->referrer = Referer::get(); // 'google.com'
             $user->save();
 
+            $userData = 'Password:'.' '.$password;
+            $setting = Setting::find(1);
+            $mail = new \App\Http\Controllers\Common\PhpMailController();
```

Comment:
Keep the imports always on top that is before the class, makes your code more readable